React/Base/RCTConvert.m

```diff
@@ -146,7 +146,7 @@ + (NSURLRequest *)NSURLRequest:(id)json
       URLString = [NSString stringWithFormat:@"%@.bundle/%@", bundleName, URLString];
     }
 
-    URL = [self NSURL:URLString];
+    URL = [self NSURL:RCTNilIfNull(URLString)];
```

Comment:
`[self NSURL]` is called from multiple places, wouldn't it be better to handle `NSNULL` inside of `NSURL`?

Reply:
Yes, I think you're right. Moved the fix in `NSURL:`.